Makefile

```diff
@@ -56,6 +56,11 @@ vet:
 check_license:
 	./scripts/check_license.sh
 
+# Turn on recursive globbing to find all shell scripts.
+check_shell:
+	bash -c "shopt -s globstar"
+	shellcheck */**/*.sh
```

Comment:
I think that you want to do:
```
shopt -s globstar && \
shellcheck */**/*.sh
```
right ?

Reply:
`make` must use the Bourne shell to execute commands:

```
~/projects/go-algorand:shellcheck$ make check_shell 
shopt -s globstar && \
        shellcheck */**/*.sh
/bin/sh: 1: shopt: not found
Makefile:63: recipe for target 'check_shell' failed
make: *** [check_shell] Error 127
```
Can we specify the shell in the `Makefile`?

Reply:
I think that we should attempt to stay more posix compliant here.. 
what's about `find . -name "*.sh" | xargs shellcheck` ?

Reply:
Ok